source/user-manual/capabilities/virustotal-scan/integration.rst

```diff
@@ -35,7 +35,13 @@ Getting started
 
 In order for this integration to function, the first thing that must be completed is the configuration of the VirusTotal integration as explained in the :doc:`Local configuration section <../../reference/ossec-conf/integration>` and as shown below:
 
-1. The following section must be added to the ``ossec.conf`` file:
+1. Install python ``requests`` module:
+
+.. code-block:: xml
+
+    pip install requests
```

Comment:
Since it must be executed as superuser we should add `#` character , right? @crolopez